docs/manual/config/layouts.rst

```diff
@@ -12,6 +12,16 @@ The first layout in the list is the default. If you define more than one
 layout, you will probably also want to define key bindings to let you switch to
 the next and previous layouts.
 
+Qtile uses python's inbuilt object serialization(pickle) for restarts.
+For custom layouts it is required, that if you have unpickelable objects 
+within your layout ,then you  have to add an exception for your layout 
```

Comment:
Aniruddh Kanojia wrote:
> > @@ -12,6 +12,16 @@ The first layout in the list is the default. If you define more than one
> >  layout, you will probably also want to define key bindings to let you switch to
> >  the next and previous layouts.
> >  
> > +Qtile uses python's inbuilt object serialization(pickle) for restarts.
> > +For custom layouts it is required, that if you have unpickelable objects 
> > +within your layout ,then you  have to add an exception for your layout 
> 
> i don't understand , what you mean by "move all commas below" ?
It has to be "<word><comma><blank><word>" instead of "<word><blank><comma><word>".
It creates issues with newlines - The comma appears in the new line instead of the
end of the old line.

> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/qtile/qtile/pull/736/files#r36577165